ecomap/src/python/ecomap/config.py

```diff
@@ -0,0 +1,52 @@
+from ConfigParser import SafeConfigParser
+from time import time
+import logging
+
+password = ['password', 'pass', 'pswd']          # keys which must be string
+logger = logging.getLogger('example')
+logging.basicConfig(level=logging.DEBUG)
+timeToUpdate = 15 * 60                           # 15 minutes
+
+
+class Config(object):
+
+    def __new__(cls, *args):
+        if not hasattr(cls, '_instance'):        # if there is not instance
+            cls._instance = object.__new__(cls)  # create it
+        return cls._instance                     # else return it
+
+    def __init__(self, path, logger):
+        self.config = {}                         # dictionary, contains configs
+        self.timeUpdate = None                   # time to update dictionary
+        self.timeCreated = time()                # create time of instance
+        self.path = path                         # path to file (temporary)
+        self.logger = logger
+        self._parseConfs()                       # get configs at start
+        self.logger.debug('Initialized instance with time created: {}'.
+                          format(self.timeCreated))
+
+    def get(self):                               # method which checks if we
+        if self.timeUpdate < time():             # need to update configs
+            self.config = {}                     # nullify configs dictionary
+            self.timeUpdate = time() + timeToUpdate  # set time to update
+            self._parseConfs()                   # parse config file
+        return self.config
+
+    def _parseConfs(self):
+        self.logger.debug('Parsed ecomap.conf at {}'.format(time()))
```

Comment:
Please use for logging next format:
self.logger.debug('Parsed ecomap.conf at %s', time.time())